services/core/OpenADRVenAgent/requirements.txt

```diff
@@ -1,4 +0,0 @@
-isodate==0.6
```

Comment:
Why are these no longer required?

Reply:
That file was deleted in error. Thanks for catching the mistake! I'll put it back.